dangerzone/gui/main_window.py

```diff
@@ -229,6 +228,8 @@ def __init__(self, dangerzone: DangerzoneGui) -> None:
                     self.handle_docker_desktop_version_check(is_version_valid, version)
             except errors.UnsupportedContainerRuntime as e:
                 pass  # It's catched later in the flow.
+            except errors.NoContainerTechException as e:
+                pass  # It's catched later in the flow.
```

Comment:
I have an `IMPROVE_ME` note in this commit, and I was wondering the following. Wouldn't it be better if we added the Docker Desktop check in this part of the code?

https://github.com/freedomofpress/dangerzone/blob/b5c09e51d8fbc3f74e5af78193f51b055ab7ebc4/dangerzone/gui/main_window.py#L620-L627

I'm suggesting this part because at this point we know that the container runtime is installed, and is up and running. This way, we won't have to swallow these exceptions. What do you think?

Reply:
I've opened a separate issue for this (https://github.com/freedomofpress/dangerzone/issues/1132), and I'll refer to it from this commit in the code. In the meantime, we'll merge this PR to not block the 0.9.0 release.